### ISSUE TITLE
Change Neighborhood heading to Assigned

### ISSUE THREAD

**Issue by jtf621** [codefordurham/school-navigator#222]
clarify from feedback in #189

**Comment by jtf621** [comment]
@sarahsba, @lemanal, and @jtf621 discussed tonight.

**Comment by jtf621** [comment]
wait for UX redesign

**Comment by mema82** [comment]
Is this waiting on something else for this to be merged into the frontend? I see its merged but not updated on live site? Thanks!

**Comment by lemanal** [comment]
Good point @mema82 , took it a little bit to make it to production, but it's up. Thanks!
